pkg/constants/constants.go

```diff
@@ -183,4 +183,28 @@ const (
 	// has completed the sync of the volume. This annotation is set by the CSI driver
 	// when the sync is completed.
 	CSIVSphereVolumeSyncAnnotationValueCompleted = "completed"
+
+	// AsyncQuotaPerformCheckAnnotationKey is the annotation key indicating that the ContentLibrary
+	// associated with a VirtualMachinePublishRequest is subject to VCFA quota approval.
+	AsyncQuotaPerformCheckAnnotationKey = "vmware-system-vcfa-storage-quota"
```

Comment:
Isn't this the CL label key? Do we need to insert this in the publish request?

Reply:
yes, good catch.